lib/cookbook-release.rb

```diff
@@ -49,15 +49,19 @@ def define_tasks(namespaced)
           opts = {
             no_prompt: ENV['NO_PROMPT'],
             category: ENV['COOKBOOK_CATEGORY'],
+            ignore_merge_commits: ENV['IGNORE_MERGE_COMMITS']
```

Comment:
I think you should make this a default. The original code already had a clear intent to remove merge commits (but no a good implementation).You don't even have to make this configurable